rush.json

```diff
@@ -573,7 +573,11 @@
     {
       "packageName": "@azure/dev-tool",
       "projectFolder": "common/tools/dev-tool",
-      "versionPolicyName": "utility"
+      "versionPolicyName": "utility",
+      // Add Identity to decoupledLocalDependencies so that dev-tool uses the package from npm, avoiding a cyclic dependency.
```

Comment:
this is good, and better than living with one version older and playing catch up.